src/call.c

```diff
@@ -1243,6 +1243,25 @@ int call_hold(struct call *call, bool hold)
 }
 
 
+/**
+ * Switch call to conference or not
+ *
+ * @param call        Call object
+ * @param conference  True for confernce, false for not
+ *
+ * @return 0 if success, otherwise errorcode
+ */
+int call_conference(struct call *call, bool conference)
+{
+	struct audio *au = call_audio(call);
+
+	if (!au)
+		return EINVAL;
+
+	return audio_set_conference(au, conference);
+}
```

Comment:
also consider using the `call_audio` function to fetch the audio object from the call object.
then there is no need for an explicit wrapper function.